aws/cluster/cluster-spec.tf

```diff
@@ -192,6 +192,7 @@ locals {
       serviceClusterIPRange  = "100.64.0.0/13"
       sshAccess              = var.ssh-cidrs
 
+      DisableSubnetTags = var.disable-subnet-tags
```

Comment:
For some reason, this option is recognised in JSON with capital "D" https://github.com/johngmyers/kops/blob/c3ce874bffa4b697f391bde08ea1d6b5e8ba88cf/pkg/apis/kops/v1alpha2/cluster.go#L197